addons/mail/static/src/new/core/thread_model.js

```diff
@@ -93,6 +93,10 @@ export class Thread {
         store.threads[this.localId] = this;
     }
 
+    get mappedMessages() {
+        return this.messages.map((id) => this._store.messages[id]);
+    }
+
```

Comment:
Would be best to rename old `messages` to `messageIds` and name this getter `messages`

Reply:
Yes but it's a bigger diff than what I wanted to do :)